Missing install-sh Script for `synfig-1.5.2.tar.gz`

While trying to do the release build for synfig-1.5.2, I found that install-sh script is missing

```
==> make install
Making install in libltdl
/bin/sh /private/tmp/synfig-20240805-72201-uuleq8/synfig-1.5.2/config/install-sh -d . libltdl
cp ./libltdl/lt__argz_.h libltdl/lt__argz.h-t
mv libltdl/lt__argz.h-t libltdl/lt__argz.h
/Applications/Xcode.app/Contents/Developer/usr/bin/make  install-am
make[2]: ./../config/install-sh: No such file or directory
make[2]: ./../config/install-sh: No such file or directory
/bin/sh ./libtool  --tag=CC   --mode=compile clang -DHAVE_CONFIG_H -I.  -DLTDLOPEN=libltdlc -DLT_CONFIG_H='<config.h>' -DLTDL -I. -I. -Ilibltdl -I./libltdl   -g -O2 -c -o libltdlc_la-lt__alloc.lo `test -f 'lt__alloc.c' || echo './'`lt__alloc.c
/bin/sh ./libtool  --tag=CC   --mode=compile clang -DHAVE_CONFIG_H -I.  -DLTDLOPEN=libltdlc -DLT_CONFIG_H='<config.h>' -DLTDL -I. -I. -Ilibltdl -I./libltdl   -g -O2 -c -o libltdlc_la-lt_dlloader.lo `test -f 'lt_dlloader.c' || echo './'`lt_dlloader.c
/bin/sh ./libtool  --tag=CC   --mode=compile clang -DHAVE_CONFIG_H -I.  -DLTDLOPEN=libltdlc -DLT_CONFIG_H='<config.h>' -DLTDL -I. -I. -Ilibltdl -I./libltdl   -g -O2 -c -o libltdlc_la-lt_error.lo `test -f 'lt_error.c' || echo './'`lt_error.c
make[2]: *** [loaders/.dirstamp] Error 1
make[2]: *** Waiting for unfinished jobs....
make[2]: *** [loaders/.deps/.dirstamp] Error 1
```

--- a/synfig-core/bootstrap.sh
+++ b/synfig-core/bootstrap.sh
@@ -50,4 +50,9 @@ sed 's/itlocaledir = $(prefix)\/$(DATADIRNAME)\/locale/itlocaledir = $(datarootd
 # -- force didn't work under MacOS
 mv -f po/Makefile.in.in.tmp po/Makefile.in.in
 
+# Fix https://github.com/synfig/synfig/issues/3398
+# For compatibility with MacOS we have to make sure "sh" binary 
+# is found in "/bin/sh", not "/usr/bin/sh"
+sed -i "s|#!/usr/bin/sh|#!/bin/sh|" config/install-sh
+
 echo "Done! Please run ./configure now."

--- a/synfig-core/bootstrap.sh
+++ b/synfig-core/bootstrap.sh
@@ -53,6 +53,8 @@ mv -f po/Makefile.in.in.tmp po/Makefile.in.in
 # Fix https://github.com/synfig/synfig/issues/3398
 # For compatibility with MacOS we have to make sure "sh" binary 
 # is found in "/bin/sh", not "/usr/bin/sh"
-sed -i "s|#!/usr/bin/sh|#!/bin/sh|" config/install-sh
+sed "s|#!/usr/bin/sh|#!/bin/sh|" < config/install-sh > config/install-sh.tmp
+mv -f config/install-sh.tmp config/install-sh
+chmod +x config/install-sh
 
 echo "Done! Please run ./configure now."

--- a/synfig-studio/bootstrap.sh
+++ b/synfig-studio/bootstrap.sh
@@ -34,4 +34,9 @@ sed 's/itlocaledir = $(prefix)\/$(DATADIRNAME)\/locale/itlocaledir = $(datarootd
 # -- force didn't work under MacOS
 mv -f po/Makefile.in.in.tmp po/Makefile.in.in
 
+# Fix https://github.com/synfig/synfig/issues/3398
+# For compatibility with MacOS we have to make sure "sh" binary 
+# is found in "/bin/sh", not "/usr/bin/sh"
+sed -i "s|#!/usr/bin/sh|#!/bin/sh|" config/install-sh
+
 echo "Done! Please run ./configure now."

--- a/synfig-studio/bootstrap.sh
+++ b/synfig-studio/bootstrap.sh
@@ -37,6 +37,8 @@ mv -f po/Makefile.in.in.tmp po/Makefile.in.in
 # Fix https://github.com/synfig/synfig/issues/3398
 # For compatibility with MacOS we have to make sure "sh" binary 
 # is found in "/bin/sh", not "/usr/bin/sh"
-sed -i "s|#!/usr/bin/sh|#!/bin/sh|" config/install-sh
+sed "s|#!/usr/bin/sh|#!/bin/sh|" < config/install-sh > config/install-sh.tmp
+mv -f config/install-sh.tmp config/install-sh
+chmod +x config/install-sh
 
 echo "Done! Please run ./configure now."

